debezium-server/debezium-server-pulsar/src/test/java/io/debezium/server/pulsar/PulsarIT.java

```diff
@@ -44,6 +44,7 @@
 
     protected static PulsarClient pulsarClient;
     @Inject
+    PulsarContainer pulsarContainer;
```

Comment:
This shouldn't be injected, but rather controlled via a lifecycle listener, similar to what we do for Postgres in `DebeziumServerIT`.

Reply:
@gunnarmorling Do we use any startup command for running pulsar when starting the container? Because I am encountering this:
```
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
Caused by: java.lang.reflect.InvocationTargetException
Caused by: java.lang.RuntimeException: Unable to start Quarkus test resource io.debezium.server.pulsar.PulsarTestResourceLifecycleManager@3b63f49e
Caused by: org.testcontainers.containers.ContainerLaunchException: Container startup failed
Caused by: org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception
Caused by: org.testcontainers.containers.ContainerLaunchException: Could not create/start container
Caused by: java.lang.IllegalStateException: Container exited with code 1
```

Reply:
And when starting the container like this using a command to start pulsar in standalone mode: 
```
    private static final GenericContainer<?> container = new GenericContainer<>(PULSAR_IMAGE)
            .waitingFor(Wait.forLogMessage(".*messaging service is ready.*\\s", 1))
            .withCommand("/pulsar/bin/pulsar", "standalone")
            .withExposedPorts(PULSAR_PORT)
            .withStartupTimeout(Duration.ofSeconds(30));
```
I am encountering something like:
```
2021-01-20 17:59:14,886 WARN  [org.apa.pul.cli.imp.ConnectionPool] (pulsar-client-io-1-1) Failed to open connection to localhost:6650 : org.apache.pulsar.shade.io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: localhost/127.0.0.1:6650
```

Reply:
We used 90 sec wait time in the original Dockerfile; perhaps you need to increase from the 30 sec you have here? We also exposed an HTTP port there, but I'm not sure whether/where that's needed. Worth trying to stick as much as possible to the original config from the Maven plug-in and see where things go wrong.